### DISCUSSION TITLE
Setting up project development- Solidity import error on text:fix

### DISCUSSION THREAD

**Discussion by StefanoGagliardi** [ethereum-ts/TypeChain#698] [Q&A] [answered]
Hi guys,
i have already used with enthusiasm Typechain and finally i decided to dedicate a few hours of my time to contribute to projects that I appreciated. I found an interesting issue and i decided to try to solve it.

I follow the "Contribute" guide with some additional installation and version downgrade.
Some test command inside packages works fine, don't works `pnpm text:fix` in root.

I get the following error:

![error-solidity](https://user-images.githubusercontent.com/45005269/169170257-4cb1627f-f877-4e9a-838b-3fe9d43d7930.PNG)
 In case this has happened to any of you before?
I installed pnpm, dependencies within the project, downgrade npm to version 6

**Comment by StefanoGagliardi** [comment, marked as answer]
Resolved with edit `scripts/compile-contracts.ts`
![resolved](https://user-images.githubusercontent.com/45005269/169172071-92f4da55-08ae-42e6-98ce-f718bc60fbdf.PNG)

I need to add commented options for work:
1. --base-path .
2. `--include-path ./contracts/${dirName}`

I have a question, "pnpm test:fix" in the root run all test of all packages?
Because the test result output is different from "Typechain" package test result

**Comment by krzkaczor** [comment]
Are u running on windows? Please don't hesitate to make a PR fix with fix :) 

> I have a question, "pnpm test:fix" in the root run all test of all packages?

yes
